packages/datatrak-web/src/features/Survey/SurveyContext/SurveyContext.tsx

```diff
@@ -71,7 +84,7 @@ export const SurveyContext = ({ children, surveyCode, countryCode }) => {
         ...screen,
         surveyScreenComponents: screen.surveyScreenComponents.filter(question => {
           // If a primary entity code is pre-set for the survey, hide the primary entity question and its ancestor questions
-          if (primaryEntityCode) {
+          if (primaryEntityCode && !isReviewScreen) {
```

Comment:
Just checking - we don't want to hide the entity questions on the review screen?

Reply:
I will double check with Georgia but I tried it both ways and I thought it was really nice to be able to see the entity that was selected. For example one of the surveys is for moving an asset to a new entity and it was a bit confusing without ever seeing the entity